src/ui.rs

```diff
@@ -34,8 +34,17 @@ pub fn render_left_view<B: Backend>(f: &mut Frame<B>, app: &mut App, area: Rect)
   render_files(f, app, chunks[1])
 }
 
-pub fn render_status<B: Backend>(f: &mut Frame<B>, _app: &mut App, area: Rect) {
-  let paragraph = Paragraph::new("This is the current status")
+pub fn render_status<B: Backend>(f: &mut Frame<B>, app: &mut App, area: Rect) {
+  let (ahead, behind) = app.branch.ahead_behind;
+
+  let name = format!(
+    "{}↑ {}↓ -> {}",
+    ahead,
+    behind,
+    app.branch.name.as_ref().unwrap_or(&"Unknown".to_string())
```

Comment:
i think colourizing the formatting of ahead/behind would be an easy task for someone to pickup after this is merge din